javascript_es6/01.a.browser-echo/README.md

```diff
@@ -6,9 +6,9 @@ After running the bot, to see it in action, visit `http://localhost:8080`.
     ```bash
```

Comment:
This entire sample needs to be removed from the \javascript_es6 folder and moved to \samples\javascript_es6\ folder.